pkg/commands/compute/compute_deploy_test.go

```diff
@@ -378,6 +378,62 @@ func TestDeploy(t *testing.T) {
 				"Deployed package (service 123, version 4)",
 			},
 		},
+		{
+			name: "success with --backend and no --backend-port",
+			args: args("compute deploy --token 123 --backend host.com"),
```

Comment:
NIT: I prefer to order flags alphabetically (even in test code) just so it's easier to scan for flags (repeat this suggestion throughout).

```suggestion
			args: args("compute deploy --backend host.com --token 123"),
```

Reply:
Done.